noah/src/main/scala/edu/uci/ics/cloudberry/noah/adm/TweetNoGeoTagException.java

```diff
@@ -0,0 +1,74 @@
+package edu.uci.ics.cloudberry.noah.adm;
+
+import edu.uci.ics.cloudberry.gnosis.USGeoGnosis;
+import twitter4j.GeoLocation;
+import twitter4j.Status;
+
+public class TweetNoGeoTagException {
```

Comment:
let change the name to not end with Exception which make it looks like a type of exception.